packages/web-pkg/src/configuration/manager.ts

```diff
@@ -14,6 +14,7 @@ export interface RawConfig {
   auth?: any
   openIdConnect?: any
   options?: OptionsConfiguration
+  logoutUrl?: string
```

Comment:
Please make the `logoutUrl` part of the `OptionsConfiguration` for now. Adding new config in oCIS is a pain, so we just go with putting everything in `options` for the moment.

Reply:
alright 👍🏼